<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <link rel="stylesheet" href="style/base.css" type="text/css" />
</head>
<body>
    <div class="document">


<div class="content">
<!--
<div style="border: 1px solid black; padding: 5px; text-align: justify; font-style: italic;"><b>LEfSe users:</b> Due to a bug outside of our control in the current version of Galaxy, Firefox and Safari users may need to manually refresh the History panel in order for workflow steps to be accurately updated.  Opera users should be unaffected, and we're hoping to find a workaround for LEfSe.  We apologize for the inconvenience, which should hopefully not interrupt normal usage.<br/>- <a href="mailto:chuttenh@hsph.harvard.edu">Curtis Huttenhower</a>, 08-15-11</div>
-->
<p><img src="http://huttenhower.sph.harvard.edu/sites/default/files/pictures/HSPH-FXB-building.jpg" width="180" height="125" alt="Francois-Xavier Bagnoud Building, Harvard School of Public Health" style="float:right;margin:5px;" /></p> 
<hr/>
<p>Thanks for visiting our lab's tools and applications page, implemented within the <a href=" http://main.g2.bx.psu.edu">Galaxy</a> web application and workflow framework.  Here, we provide a number of resources for metagenomic and functional genomic analyses, intended for research and academic use.  Please see the menus and folders to the left for an overview of available tools including documentation, sample data, and publications.
</b> 

<p>
Our lab's research interests include metagenomics and the <a href="http://www.hmpdacc.org">human microbiome</a>, the relationships between microbial communities and human health, microbiome systems biology, and large-scale computational methods for studying all of these areas. In addition to the tools provided here, feel free to take a look at our additional <a href="http://huttenhower.sph.harvard.edu/research">research</a> and <a href="http://huttenhower.sph.harvard.edu/publications">publications</a>, including the <a href="http://huttenhower.sph.harvard.edu/getting-started-sleipnir">Sleipnir library</a> for computational functional genomics.
</b>


<p>
The tools are available here without account creation. However, you are strongly invited to create an account for having access to the history, saved analyses, datasets and workflows. You can create an account and/or log in using the User menu in the top-right corner.
</b>

<p>
If you have any comments, questions, or suggestions, please contact <a href="http://huttenhower.sph.harvard.edu/contact">Dr. Huttenhower</a>.
</b>
<!--
<div style="border: 1px solid black; padding: 5px; text-align: justify; font-style: italic;"><b>LEfSe users:</b> Note that due to LEfSe's recent popularity, we temporarily overwhelmed our available IT resources.  LEfSe was experiencing periodic outages as a result.  Many thanks for your interest - as far as things go, it's not a bad problem to have!  Our IT team has successfully restored full functionality, and we apologize if you were affected by the outage.  As always, please contact us directly if you have questions or would like to try out a copy of the command-line-only LEfSe Python implementation.<br/>- <a href="mailto:chuttenh@hsph.harvard.edu">Curtis Huttenhower</a>, 08-18-11</div>
 -->
</body>
</html>

